contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -1163,7 +1163,7 @@ define_custom_variables(void)
 							gettext_noop("Temp oid buffer size"),
 							NULL,
 							&temp_oid_buffer_size,
-							0, 0, 131072,
+							65536, 0, 131072,
```

Comment:
this seems out of place for this change. Is this expected?

Reply:
this is for testing purposes only. This will be reverted before the PR gets merged.

Reply:
won't it cause the tests to fail? can we write the tests in a way that is independent of the actual GUC value set?

Reply:
Reverting this won't cause the tests to fail, since there won't be any lock contention anyways.

Reply:
leaving this in the merge bc with this fix, we can reenable temp oid for testing